packages/core/src/store/anchor-request-store.ts

```diff
@@ -73,7 +75,34 @@ export class AnchorRequestStore extends ObjectStore<StreamID, AnchorRequestData>
     } while (true)
   }
 
-  async close(): Promise<void> {
-    await this.store.close(this.useCaseName)
+  async *infiniteList(
+    batchSize = 1,
+    batchDelay = 100
```

Comment:
this delay doesn't actually happen between each batch, it happens between each full loop when you get to the end of the entire result set and are about to loop back around to the beginning.  Can we rename it so that's clearer?  Maybe `restartDelay` or something like that?

Reply:
let it be `restartDelay`